usaspending_api/recipient/management/sql/restock_recipient_profile.sql

```diff
@@ -20,6 +20,12 @@ CREATE MATERIALIZED VIEW public.temporary_recipients_from_transactions_view AS (
     ))::uuid AS recipient_hash,
     COALESCE(fpds.awardee_or_recipient_uniqu, fabs.awardee_or_recipient_uniqu) AS recipient_unique_id,
     COALESCE(fpds.ultimate_parent_unique_ide, fabs.ultimate_parent_unique_ide) AS parent_recipient_unique_id,
+    MD5(UPPER(
+      CASE
+        WHEN COALESCE(fpds.ultimate_parent_uei, fabs.ultimate_parent_uei) IS NOT NULL THEN CONCAT('uei-', COALESCE(fpds.ultimate_parent_uei, fabs.ultimate_parent_uei))
+        WHEN COALESCE(fpds.ultimate_parent_unique_ide, fabs.ultimate_parent_unique_ide) IS NOT NULL THEN CONCAT('duns-', COALESCE(fpds.ultimate_parent_unique_ide, fabs.ultimate_parent_unique_ide))
+        ELSE CONCAT('name-', COALESCE(fpds.ultimate_parent_legal_enti, fabs.ultimate_parent_legal_enti, '')) END
+    ))::uuid AS parent_recipient_hash,
```

Comment:
Do you think this could have a measurable impact on performance? Adding many minutes to the recipient_profile build out? It would be good to get some comparative before/after test-timings on this with a production sized dataset in sandbox or qat.

Reply:
removed this hash